### ISSUE TITLE
Show pending tips amount in Rewards wallet card summary

### ISSUE THREAD

**Issue by zenparsing** [brave/brave-browser#17869]
Rewards panel 2.0 designs do not currently include a "pending tips" amount in the summary view. Since the 1.0 designs did include this information, consider adding it back.

**Comment by GeetaSarvadnya** [comment]
Reproduced the issue in `1.31.41` nightly build

**Comment by Miyayes** [comment]
![image](https://user-images.githubusercontent.com/11497541/133471873-5843ae94-99dc-4771-afe4-c20d229706c4.png)

![image](https://user-images.githubusercontent.com/11497541/133471892-ba102289-4809-4ec7-aa14-dbe167b468c3.png)

**Comment by Miyayes** [comment]
New design: 

![image](https://user-images.githubusercontent.com/11497541/133689802-ec41f244-7b34-47b6-8a72-bdeb54d0e211.png)

**Comment by LaurenWags** [comment]
labelling as `QA/Blocked` until uplift to 1.31.x is completed

**Comment by LaurenWags** [comment]
Removing `QA/Blocked`, this has been merged to 1.31.x and will be available for testing in 1.31.69 and above

**Comment by LaurenWags** [comment]
Verification in progress using 

```
Brave | 1.31.71 Chromium: 94.0.4606.71 (Official Build) beta (x86_64)
-- | --
Revision | 1d32b169326531e600d836bd395efc1b53d0f6ef-refs/branch-heads/4606@{#1256}
OS | macOS Version 10.15.7 (Build 19H1417)
```

<details>
<summary>Clean profile</summary>

Verified test plan from https://github.com/brave/brave-core/pull/10182#issue-1003282413

Confirmed link to pending tips only shows on brave://rewards page and does not show when panel is accessed from BAT logo in URL bar.
Confirmed "see all" link opens the pending tips modal.
Confirmed pending tips can still be removed from the list.
Confirmed balance of pending tips on wallet panel updates as tips are added/removed.

Example | Example | Example | Example
----- | ----- | ----- | -----

</details>

<details>
<summary>Upgrade profile</summary>

Used 1.30.87 (current release) to set up some pending tips.

Example | Example
----- | ----- 

Renamed profile to `Brave-Browser-Beta` and launched with 1.31.71.
Confirmed link to pending tips only shows on brave://rewards page and does not show when panel is accessed from BAT logo in URL bar.
Confirmed "see all" link opens the pending tips modal.
Confirmed pending tips can still be removed from the list.
Confirmed balance of pending tips on wallet panel updates as tips are added/removed.

Example | Example | Example | Example | Example | Example
----- | ----- | ----- | ----- | ----- | -----

</details>
